src/parser/generated_parser.ts

```diff
@@ -3383,7 +3387,13 @@ export class GeneratedParser extends Parser {
             (literal = this.expect(",")) &&
             ((opt = this._tmp_125()), 1)
         ) {
-            return RAISE_SYNTAX_ERROR_KNOWN_LOCATION(a, "Generator expression must be parenthesized");
+            return pegen.raise_error_known_location(
+                this,
+                pySyntaxError,
+                a.lineno,
+                a.col_offset + 1,
```

Comment:
this isn't quite correct. Sometimes `a` is an AST sometimes it's a `TokenInfo`. We could maybe add getters to `TokenInfo` for `lineno` and `col_offset`?